src/util/instanceFilter.tsx

```diff
@@ -2,7 +2,7 @@ import { LxdInstanceStatus } from "types/instance";
 
 export interface InstanceFilters {
   queries: string[];
-  statuses: LxdInstanceStatus[];
+  statuses: string[];
```

Comment:
I don't understand this change. We have a limited (and known) range of possible values for this, expressed by the `LxdInstanceStatus`. Why not using it?

Reply:
The filters are stored in the url, which is string only. This change avoids casting to and from string, when translating the statuses to and from the searchparameters. Because the url parameters can't be typed as status.

Reply:
I can let this go, but I still see it as a step backwards.

Reply:
Actually we can keep the status type as well. Through the simplifications, there is just a single cast left.